### DISCUSSION TITLE
How to pass parameters when removing duplicate elements when extracting the table

### DISCUSSION THREAD

**Discussion by situchen** [jsvine/pdfplumber#378] [Q&A] [answered]
page.extract_tables() How to pass parameters can directly call dedupe_chars() method？

**Comment by samkit-jain** [comment]
Hi @situchen Could you please elaborate on what you are trying to achieve with an example?

**Reply by samkit-jain** [reply]
@situchen I see that you left a comment on one of the dedupe related commits at https://github.com/jsvine/pdfplumber/commit/b132d45db553bbb68a2bbda3c3efb88b53f6fd3d#commitcomment-48354718. Does that resolve your query?

**Comment by jsvine** [comment, marked as answer]
Hi @situchen. I agree with @samkit-jain — it is difficult to answer this question confidently without having more details. But I think you may be looking for this:

```python
original_page = my_pdf.pages[0]
deduped_page = original_page.dedupe_chars()
tables = deduped_page.extract_tables()
```
